lib/engines/content_block_manager/app/services/content_block_manager/get_host_content_items.rb

```diff
@@ -4,18 +4,22 @@
 
 module ContentBlockManager
   class GetHostContentItems
-    attr_reader :content_id
+    attr_reader :content_id, :page, :order
 
-    def initialize(content_id:)
+    DEFAULT_ORDER = "-unique_pageviews".freeze
+
+    def initialize(content_id:, page: nil, order: nil)
       self.content_id = content_id
+      self.page = page
```

Comment:
minor question: is having the `self` attributes here a more old fashioned way of having class attributes, over the `@` ones?

Reply:
It's more akin to having an `@` variable with an `attr_reader`, which I kinda prefer and is more Rubyish to my eyes. Think I'll get into the habit of removing this pattern when I see it.